### PR TITLE
Fix avatar not being null when it's null

### PR DESCRIPTION
**Pull request by ArsenArsen**
### Prerequisites
* [x] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [x] This pull request follows the code style of the project
* [ ] I have tested this feature thoroughly
  * [Proof of testing](http://nope/)

**Issues Fixed:** getAvatar() would return an invalid avatar instead of `null`

### Changes Proposed in this Pull Request
* Add an if check

### CODE REVIEW

**Comment by ArsenArsen** [comment]
Okay, tomorrow :)

File: src/main/java/sx/blah/discord/handle/impl/obj/User.java [hunk 1]
```diff
@@ -38,6 +38,17 @@
 public class User implements IUser {
 
 	/**
+	 * Default avatars for users
+	 */
+	private static String[] DEFAULT_AVATARS = {
```
Comment (austinv11) [inline src/main/java/sx/blah/discord/handle/impl/obj/User.java]: I don't know if I like this hard coding

Comment (ArsenArsen) [inline src/main/java/sx/blah/discord/handle/impl/obj/User.java]: How else is it supposed to work? I could make it an enum possibly?

Comment (austinv11) [inline src/main/java/sx/blah/discord/handle/impl/obj/User.java]: Pointing to/hardcoding internal assets is iffy. They can often be relocated as has happened in the past.

Comment (ArsenArsen) [inline src/main/java/sx/blah/discord/handle/impl/obj/User.java]: Still, how else would it be possible? As far as my awareness goes Discord payloads don't contain a default avatar field.

**Comment by austinv11** [comment]
Test your commit please. A

**Comment by austinv11** [comment]
oops

File: src/main/java/sx/blah/discord/api/internal/DiscordEndpoints.java [hunk 2]
```diff
@@ -138,4 +138,9 @@
 	 */
 	public static final String REACTIONS_USER = REACTIONS_USER_LIST + "/%s";
 
+	/**
+	 * Default avatars (arg 1 is <code>discrim % 5</code>)
+	 */
+	public static final String DEFAULT_AVATAR = BASE + "https://cdn.discordapp.com/embed/avatars/%d.png";
```
Comment (austinv11) [inline src/main/java/sx/blah/discord/api/internal/DiscordEndpoints.java]: Test your PR please

**Pull request merged**